main.js

```diff
@@ -222,8 +213,8 @@ async function _checkStatus({
   // get JSON RevocationList
```

Comment:
There's revocation list language in the comments here -- is that still proper?

Reply:
Removed [b9a53d6](https://github.com/digitalbazaar/vc-status-list/pull/6/commits/b9a53d64e2c4ae814291ee6aae9a9cd26d3758c6)